### PR TITLE
Pass DEB_BUILD_PROFILES through to the build

### PR DESCRIPTION
**Pull request by jamessan**

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -330,9 +331,9 @@ fi
 
 Info "Running build"
 # shellcheck disable=SC2086
-docker run --env=DEB_BUILD_OPTIONS="${DEB_BUILD_OPTIONS:-}" ${ARGS} "${TAG}"
+docker run --env=DEB_BUILD_OPTIONS="${DEB_BUILD_OPTIONS:-}" --env=DEB_BUILD_PROFILES="${DEB_BUILD_PROFILES:-}" ${ARGS} "${TAG}"
 
-Info "Copying build artefacts to ${TRAVIS_DEBIAN_TARGET_DIR}"
+Info "Copying build artifacts to ${TRAVIS_DEBIAN_TARGET_DIR}"
```
Comment (lamby) [inline script.sh]: en_GB here... Please drop this bit ;)

**Review by lamby — COMMENTED** [review]
Thanks for this, just waiting on …

**Comment by jamessan** [comment]
Done.

**Comment by lamby** [comment]
Thanks @jamessan !

**Pull request merged**